addons/mail/static/tests/qunit_suite_tests/chatter_tests.js

```diff
@@ -262,6 +262,123 @@ QUnit.test('list activity exception widget with activity', async function (asser
     list.destroy();
 });
 
+QUnit.test('list activity widget: done the activity with "ENTER" keyboard shortcut', async function (assert) {
+    assert.expect(1);
+
+    const pyEnv = await startServer();
+    const mailActivityTypeId1 = pyEnv['mail.activity.type'].create();
+    const mailActivityId1 = pyEnv['mail.activity'].create([
+        {
+            display_name: "Call with Al",
+            date_deadline: moment().format("YYYY-MM-DD"), // now
+            can_write: true,
+            state: "today",
+            user_id: pyEnv.currentUserId,
+            create_uid: pyEnv.currentUserId,
+            activity_type_id: mailActivityTypeId1,
+        }
+    ]);
+    pyEnv['res.users'].write([pyEnv.currentUserId], {
+        activity_ids: [mailActivityId1],
+        activity_state: 'today',
+        activity_summary: 'Call with Al',
+        activity_type_id: mailActivityTypeId1,
+    });
+
+    const { widget: list } = await start({
+        hasView: true,
+        View: ListView,
+        model: 'res.users',
+        arch: `
+            <list>
+                <field name="activity_ids" widget="list_activity"/>
+            </list>`,
+        mockRPC: function (route, args) {
+            if (args.method === 'action_feedback') {
+                pyEnv['res.users'].write([pyEnv.currentUserId], {
+                    activity_ids: [],
+                    activity_state: '',
+                });
+                return Promise.resolve();
+            }
```

Comment:
Can you implement `action_feedback` in `MockServer` so it does correct change of data whatever the state of server?